python/examples/advanced/Streaming_Data_with_Log_Rotation.ipynb

```diff
@@ -0,0 +1,332 @@
+      "\u001B[0;32m~/opt/anaconda3/lib/python3.9/os.py\u001B[0m in \u001B[0;36mmakedirs\u001B[0;34m(name, mode, exist_ok)\u001B[0m\n\u001B[1;32m    223\u001B[0m             \u001B[0;32mreturn\u001B[0m\u001B[0;34m\u001B[0m\u001B[0;34m\u001B[0m\u001B[0m\n\u001B[1;32m    224\u001B[0m     \u001B[0;32mtry\u001B[0m\u001B[0;34m:\u001B[0m\u001B[0;34m\u001B[0m\u001B[0;34m\u001B[0m\u001B[0m\n\u001B[0;32m--> 225\u001B[0;31m         \u001B[0mmkdir\u001B[0m\u001B[0;34m(\u001B[0m\u001B[0mname\u001B[0m\u001B[0;34m,\u001B[0m \u001B[0mmode\u001B[0m\u001B[0;34m)\u001B[0m\u001B[0;34m\u001B[0m\u001B[0;34m\u001B[0m\u001B[0m\n\u001B[0m\u001B[1;32m    226\u001B[0m     \u001B[0;32mexcept\u001B[0m \u001B[0mOSError\u001B[0m\u001B[0;34m:\u001B[0m\u001B[0;34m\u001B[0m\u001B[0;34m\u001B[0m\u001B[0m\n\u001B[1;32m    227\u001B[0m         \u001B[0;31m# Cannot rely on checking for EEXIST, since the operating system\u001B[0m\u001B[0;34m\u001B[0m\u001B[0;34m\u001B[0m\u001B[0m\n",
+      "\u001B[0;31mFileExistsError\u001B[0m: [Errno 17] File exists: '/Users/melanie/Dev/whylogs/python/examples/advanced/example_output'"
+     ]
+    }
+   ],
+   "source": [
+    "import pandas as pd\n",
+    "!pip install psutil\n",
+    "!pip install whylogs\n",
+    "\n",
+    "import time\n",
+    "import whylogs as why\n",
+    "\n",
+    "import os\n",
+    "from os import listdir\n",
+    "from os.path import isfile\n",
+    "\n",
+    "tmp_path = os.path.join(os.getcwd(), \"example_output\")\n",
+    "\n",
+    "if not os.path.isdir(tmp_path):\n",
+    "    os.makedirs(tmp_path)"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Here is a super simple function to see the amount of files that are here before and after the logging."
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "10\n"
+     ]
+    }
+   ],
+   "source": [
+    "def count_files(tmp_path):\n",
```

Comment:
I think this works only if you have the folder already created. Maybe running a mkdir first?

Reply:
I do run mkdir up at the top when creating a the tmp_Path

Reply:
I'm sorry, it seems I pulled an outdated version. I think you're already doing that